### DISCUSSION TITLE
Adjustable font sizes

### DISCUSSION THREAD

**Discussion by jdtinsley88** [mattieserver/netbox-topology-views#202] [Q&A]
Anyway to adjust the font sizes of the device labels as well as any other text in this plugin?

I am trying to use Google Chrome to find the right CSS file and edit it, but not having any luck.

**Comment by mattieserver** [comment]
Hi,

For the moment you cannot. The plugin uses the html 'canvas' so you cannot change the values in CSS.
I might add a way to override the font size
